CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - IAM Authn bug fix - Take rexml gem to production configuration [#2493](https://github.com/cyberark/conjur/pull/2493)
+- Bug fix that removes stale puma pidfile[#2381](https://github.com/cyberark/conjur/issues/2381)
```

Comment:
Couple of things here:
- Ideally, a "fixed" changelog entry should contain both the problem and what was changed to fix it.
- A small formatting note, there should be space before the PR link.
- The link should be to the PR containing the change, rather than the issue.

This would come together as something like:
```markdown
- Previously, a stale puma pid file would prevent the Conjur server from starting 
  successfully. Conjur now removes a stale pid file at startup, if it exists.
  [#2498](https://github.com/cyberark/conjur/pull/2498)
```